flink-scala-shell/src/main/java/org/apache/flink/api/java/JarHelper.java

```diff
@@ -71,144 +71,151 @@ public JarHelper() {}
 		throws IOException {
-	if (mVerbose) { System.out.println("checking " + dirOrFile2jar);}
-	if (dirOrFile2jar.isDirectory()) {
-		String[] dirList = dirOrFile2jar.list();
-		String subPath = (path == null) ? "" : (path+dirOrFile2jar.getName()+SEP);
-		if (path != null) {
-		JarEntry je = new JarEntry(subPath);
-		je.setTime(dirOrFile2jar.lastModified());
-		jos.putNextEntry(je);
-		jos.flush();
-		jos.closeEntry();
-		}
-		for (int i = 0; i < dirList.length; i++) {
-		File f = new File(dirOrFile2jar, dirList[i]);
-		jarDir(f,jos,subPath);
-		}
-	} else if (dirOrFile2jar.exists()) {
-		if (dirOrFile2jar.getCanonicalPath().equals(mDestJarName))
-		{
-		if (mVerbose) {System.out.println("skipping " + dirOrFile2jar.getPath());}
-		return;
-		}
-
 		if (mVerbose) {
-			System.out.println("adding " + dirOrFile2jar.getPath());
+			System.out.println("checking " + dirOrFile2jar);
 		}
-		FileInputStream fis = new FileInputStream(dirOrFile2jar);
-		try {
-		JarEntry entry = new JarEntry(path+dirOrFile2jar.getName());
-		entry.setTime(dirOrFile2jar.lastModified());
-		jos.putNextEntry(entry);
-		while ((mByteCount = fis.read(mBuffer)) != -1) {
-			jos.write(mBuffer, 0, mByteCount);
-			if (mVerbose) { System.out.println("wrote " + mByteCount + " bytes");}
-		}
-		jos.flush();
-		jos.closeEntry();
-		} catch (IOException ioe) {
-		throw ioe;
-		} finally {
-		fis.close();
+		if (dirOrFile2jar.isDirectory()) {
+			String[] dirList = dirOrFile2jar.list();
+			String subPath = (path == null) ? "" : (path + dirOrFile2jar.getName() + SEP);
+			if (path != null) {
+				JarEntry je = new JarEntry(subPath);
+				je.setTime(dirOrFile2jar.lastModified());
+				jos.putNextEntry(je);
+				jos.flush();
+				jos.closeEntry();
+			}
+			for (int i = 0; i < dirList.length; i++) {
+				File f = new File(dirOrFile2jar, dirList[i]);
+				jarDir(f, jos, subPath);
+			}
+		} else if (dirOrFile2jar.exists()) {
+			if (dirOrFile2jar.getCanonicalPath().equals(mDestJarName)) {
+				if (mVerbose) {
+					System.out.println("skipping " + dirOrFile2jar.getPath());
+				}
+				return;
+			}
+
+			if (mVerbose) {
+				System.out.println("adding " + dirOrFile2jar.getPath());
+			}
+			FileInputStream fis = new FileInputStream(dirOrFile2jar);
+			try {
+				JarEntry entry = new JarEntry(path + dirOrFile2jar.getName());
+				entry.setTime(dirOrFile2jar.lastModified());
+				jos.putNextEntry(entry);
+				while ((mByteCount = fis.read(mBuffer)) != -1) {
+					jos.write(mBuffer, 0, mByteCount);
+					if (mVerbose) {
+						System.out.println("wrote " + mByteCount + " bytes");
+					}
+				}
+				jos.flush();
+				jos.closeEntry();
+			} catch (IOException ioe) {
+				throw ioe;
+			} finally {
+				fis.close();
+			}
 		}
 	}
-	}
 
 	// for debugging
 	public static void main(String[] args)
-		throws IOException
-	{
-	if (args.length < 2)
-	{
-		System.err.println("Usage: JarHelper jarname.jar directory");
-		return;
-	}
+		throws IOException {
```

Comment:
Will do. Since this was copied code I had simply done an IntelliJ code reformat.